flexget/plugins/notifiers/slack.py

```diff
@@ -35,7 +36,74 @@ class SlackNotifier(object):
             'channel': {'type': 'string'},
             'username': {'type': 'string', 'default': 'Flexget'},
             'icon_emoji': {'type': 'string'},
-            'icon_url': {'type': 'string', 'format': 'url'}
+            'icon_url': {'type': 'string', 'format': 'url'},
+            'attachments': {
```

Comment:
Attachments schema should have some required fields according to slack docs if I remember correctly.